packages/core/contracts/optimistic-assertor/implementation/OptimisticAssertor.sol

```diff
@@ -97,15 +97,17 @@ contract OptimisticAssertor is Lockable, OptimisticAssertorInterface, Ownable {
             expirationTime: block.timestamp + liveness
         });
 
-        // Check if the Sovereign Security Manager is configured to arbitrate via DVM. Note that this call will revert
-        // if the Sovereign Security Manager is configured to not allow this assertion, such as if the manager has a
-        // configured whitelist and the asserter is not on it.
-        assertions[assertionId].respectDvmOnArbitration = _checkIfShouldRespectDvmOnArbitrate(assertionId);
+        SovereignSecurityManagerInterface.AssertionPolicies memory assertionPolicies =
```

Comment:
this will fail if sovereignSecurityManager is not set.

maybe move the logic of getting assertionPolicies in internal method that also returns default policies if there is no SSM